src/xcode/ENA/ENA/Source/Models/Exposure/WarnOthersNotificationsTimer.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+
+/// Defines two timer values for the first and the second notification to be shown. The values are meant to be seconds.
+enum WarnOthersNotificationsTimer: Int {
+	
+	case timerOneTime = 7200
+	
+	case timerTwoTime = 14400
```

Comment:
Should be called `timeIntervalOne` and `timeIntervalTwo`

Reply:
And this could be just a static let, that way you wouldn't have to access it using `.rawValue`

Reply:
Done and static 👍